rpc_jobs/standard_job_checkmarx.yml

```diff
@@ -0,0 +1,57 @@
+- job-template:
+    CRON: "{CRON_WEEKLY}"
+    RPC_GATING_BRANCH: master
+    NUM_TO_KEEP: 30
+    READ_ONLY_TEST: true
+    SLAVE_TYPE: "shared"
```

Comment:
Why use a shared slave?

Reply:
Because it stays within the workding directory, and thats the fastest/most resource efficient node type for well behaved jobs.